### ISSUE TITLE
Bloop's homebrew formula shouldn't install coursier

### ISSUE THREAD

**Issue by Duhemm** [scalacenter/bloop#250]
Currently, we download Coursier and use it to build Bloop when installing via homebrew. The `coursier` binary that we download is not removed, and is installed along with the bloop. If the user already has coursier installed via homebrew, homebrew will tell the user to unlink coursier and link bloop.

Our homebrew formula should simply depend on coursier?

**Comment by olafurpg** [comment]
You can either depend on the `coursier` formula or `wget` the slim bootstrap script for single use and then delete it https://github.com/coursier/coursier/blob/master/coursier

**Comment by olafurpg** [comment]
> Our homebrew formula should simply depend on homebrew?

What do you mean?

I think it's possible to depend on another formula. Alternatively, you can `wget` the slim bootstrap script and delete it afterwards https://github.com/coursier/coursier/blob/master/coursier

**Comment by Duhemm** [comment]
> >Our homebrew formula should simply depend on homebrew?
> 
> What do you mean?

Whoops, I meant depend on coursier. I fixed it, thanks.

It's indeed possible to depend on other formulas. For instance, we already depend on the python formula.

**Comment by laughedelic** [comment]
Moreover you can depend on it only in while building/installing your formula ([docs](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#specifying-other-formulae-as-dependencies)):

```
  depends_on "python" => :build
  depends_on "coursier" => :build
```

**Comment by Duhemm** [comment]
Fixed #266
